k8s/esp_echo_gke_ingress.yaml

```diff
@@ -40,7 +40,7 @@ spec:
     spec:
       containers:
       - name: esp
-        image: b.gcr.io/endpoints/endpoints-runtime:1.0
+        image: b.gcr.io/endpoints/endpoints-runtime:latest
```

Comment:
We will need to go over and change all images in the samples. Probably do it in another commit?